sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md

```diff
@@ -99,23 +99,24 @@ The Azure.Security.KeyVault.Secrets package supports synchronous and asynchronou
 
 The following section provides several code snippets using the [above created](#create-secretclient) `client`, covering some of the most common Azure Key Vault secret service related tasks:
 
-### Async examples
+### Sync examples
```

Comment:
As discussed offline, I think the Async samples should be highlighted samples.

Reply:
I agree. But we've been told to use sync for now. I will start a thread with Krzysztof et. al.